shared/haml/user_header.haml

```diff
@@ -1,13 +1,16 @@
--# Locals: current_user, loc_prefix, [session_pairings]
+-# Locals: current_user, loc_prefix, [session_pairings], level
 :ruby
   require 'cdo/create_header'
   user_under_13 = request.cookies["_user_type_#{rack_env}"] == "student_y"
+  level = defined?(local_assigns) ?
+    local_assigns[:level] :
+    false
```

Comment:
unless I'm missing something, I don't think you need this statement -- `level` should be implicitly set to `local_assigns[:level]` by rails.

If that turns out not to be the case, I'd recommend expressing this using the safe navigation operator as `level = local_assigns&.[](:level)` instead.

Reply:
On Pegasus, I still get a undefined local variable or method `local_assigns' error with this suggestion because the user_header_option I'm interested in is coming from the _header file defined in dashboard. I think in this case, I need to explicitly check if `local_assigns' exists before trying to retrieve the level.